lib/AST/Module.cpp

```diff
@@ -1535,31 +1535,36 @@ void ModuleDecl::ImportCollector::collect(
 }
 
 static void
-collectExportedImports(const ModuleDecl *module,
+collectExportedImports(const ModuleDecl *topLevelModule,
```

Comment:
This entire function is also just duplicating existing code in lib/AST/ImportCache.cpp.